Framework/Json/inc/MantidJson/Json.h

```diff
@@ -6,21 +6,19 @@
 // SPDX - License - Identifier: GPL - 3.0 +
 #pragma once
 
-#include "MantidKernel/DllConfig.h"
+#include "MantidJson/DllConfig.h"
 
 #include <json/reader.h>
 #include <json/value.h>
 #include <json/writer.h>
 #include <string>
 
 namespace Mantid {
-namespace Kernel {
 namespace JsonHelpers {
```

Comment:
Nit picky: Can we just do `Mantid::Json` for consistency with the other libraries?

Also, in C++17 we an do nested namespaces using colons. Let's use that here to encourage it in new code:
```
namespace Mantid::Json {
}
```

Reply:
Well I'd liked to but during resolution of namespaces for Json::Value it causes issues, I'd have liked to.